packages/experiments/src/components/FolderCover/initializeFolderCovers.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { registerIcons, IIconOptions } from '@uifabric/styling';
 
-const ASSET_CDN_BASE_URL = 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/foldericons';
+const ASSET_CDN_BASE_URL = 'https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/foldericons-fluent';
```

Comment:
Host apps may already override the base URL to a different subfolder using `initializeFolderCovers()`.

Reply:
Does look like this is resolved with a revert.

Reply:
good catch! updated!